server/lib/nl/nl_page_config_next.py

```diff
@@ -18,7 +18,7 @@
   SubjectPageConfig, Tile
 from lib.nl.nl_utterance import Utterance, ChartType, ChartSpec
 from lib.nl.nl_detection import Place, RankingType
-from lib.nl.nl_utils import get_sv_name
+from lib.nl import nl_utils
```

Comment:
Shall we remove all the "nl_" file prefix under nl folder?

Reply:
Let me do in follow-up PR?  I want to rename nl_data_spec to fulfillment too.